sdk/tables/azure-tables/src/print.browser.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
You don't seem to be using this file or print.ts. May be a good idea to remove them